code/game/turfs/closed.dm

```diff
@@ -20,19 +20,20 @@
 
 
 
-
 /turf/closed/indestructible/splashscreen
 	name = "Space Station 13"
 	icon = 'icons/misc/fullscreen.dmi'
-	icon_state = "title"
+	icon_state = null
```

Comment:
I wasn't aware it could have a default, I thought it had to be null, will change soon.